### PR TITLE
all: support multiple slashes in object and caveat names

### PR DESCRIPTION
**Pull request by jakedt**

### CODE REVIEW

File: internal/services/integrationtesting/testconfigs/multipleslashes.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+schema: |+
+    definition org/identity_team/user {}
+
+    definition org/identity_team/group {
+    	relation member: org/identity_team/user | org/identity_team/group#member
+    }
+
+    definition org/docs_team/resource {
```
[the remark this answers is not in the document]
Comment (jakedt) [reply internal/services/integrationtesting/testconfigs/multipleslashes.yaml:9]: `user` is under `identity_team`, are you looking for something additional?

File: pkg/schemadsl/compiler/translator.go [hunk 2]
```diff
@@ -24,7 +24,7 @@ type translationContext struct {
 
 func (tctx translationContext) prefixedPath(definitionName string) (string, error) {
 	var prefix, name string
-	if err := stringz.SplitExact(definitionName, "/", &prefix, &name); err != nil {
+	if err := stringz.SplitInto(definitionName, "/", &prefix, &name); err != nil {
```
[the remark this answers is not in the document]
Comment (jakedt) [reply pkg/schemadsl/compiler/translator.go:27]: I agree, that _would have_ been nice

File: internal/services/integrationtesting/testconfigs/multipleslashes.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (josephschorr) [reply internal/services/integrationtesting/testconfigs/multipleslashes.yaml:9]: Yeah, I want the same resource "name" used twice, with different prefix paths

**Review by josephschorr — DISMISSED** [review]
LGTM

**Review by josephschorr — APPROVED** [review]

**Pull request merged**